ocs_ci/ocs/cluster_load.py

```diff
@@ -0,0 +1,278 @@
+"""
+A module for cluster load related functionalities
+
+"""
+import logging
+import time
+
+import ocs_ci.ocs.constants as constant
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import (
+    CephCluster, get_osd_pods_memory_sum, get_percent_used_capacity
+)
+
+
+class ClusterLoad:
+    """
+    A class for cluster load functionalities
+
+    """
+
+    def __init__(self, propagate_logs=True):
+        """
+        Initializer for ClusterLoad
+
+        Args:
+            propagate_logs (bool): True for logging, False otherwise
+
+        """
+        self.logger = logging.getLogger(__name__)
+        self.logger.propagate = propagate_logs
+        self.cl_obj = CephCluster()
+
+    def reach_cluster_load_percentage_in_throughput(
+        self, pvc_factory, pod_factory, target_percentage=0.3, cluster_limit=None
+    ):
+        """
+        This function determines how many pods, that are running FIO, needed in order to
```

Comment:
can you reword the sentence? is it adjusting the fio or the number of pods or both?

Reply:
Ack